ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -611,6 +611,21 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
       PROVIDER_IMAGE: cluster-api-provider-agent
     workflow: assisted-baremetal-operator-capi
+- as: e2e-ai-operator-disconnected-capi
```

Comment:
Be aware it's a presubmit job, not a periodic one
I'd also add the resubmit one to make it easier to debug the test if needed

Reply:
Great point, thanks Osher! I'll add that